src/hooks/custom-signature/useEthProvider.ts

```diff
@@ -1,13 +1,30 @@
-import { ref, watchEffect } from 'vue';
+import { SupportWallet } from 'src/config/wallets';
+import { useStore } from 'src/store';
+import { ref, watchEffect, computed } from 'vue';
 import { EthereumProvider } from '../types/CustomSignature';
 
 export function useEthProvider() {
   const ethProvider = ref<EthereumProvider>();
 
+  const store = useStore();
+  const currentWallet = computed(() => store.getters['general/currentWallet']);
```

Comment:
Adding custom tokens is not working in both MetaMask and Talisman(EVM).
![image](https://user-images.githubusercontent.com/92044428/174631043-145270c9-6e2a-416e-b26b-10430750327a.png)

Reply:
This shouldn't happen, do you have any idea why the `store` here is undefined?